pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "rubin-sim"
+description = "Scheduler, survey strategy analysis, and other simulation tools for Rubin Observatory."
+license = { text = "GPL" }
+classifiers = [ 
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved ::  GNU General Public License v3 or later (GPLv3+)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Topic :: Scientific/Engineering :: Astronomy",
+ ]
+urls = {documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst/rubin_sim" }
+dynamic = [ "version" ]
+
+[project.scripts]
+rs_download_data = "rubin_sim.data.rs_download_data:rs_download_data"
+rs_download_sky = "rubin_sim.data.rs_download_sky:rs_download_sky"
+addRun = "rubin_sim.maf.db.add_run:addRun"
+ddf_dir = "rubin_sim.maf.ddf_dir:ddf_dir"
+gather_summaries = "rubin_sim.maf.runComparison.gather_summaries:gather_summaries"
+generate_ss = "rubin_sim.maf.generate_ss:generate_ss"
+glance_dir = "rubin_sim.maf.glance_dir:glance_dir"
+maf_night_report = "rubin_sim.maf.maf_night_report:maf_night_report"
+metadata_dir = "rubin_sim.maf.metadata_dir:metadata_dir"
+resultdb_combine = "rubin_sim.maf.runComparison.resultdb_combine:resultdb_combine"
```

Comment:
I think we can drop this one - I thought it had already been removed from the bin directory. It does the same thing as gather_summaries, but without as many options.